ui_framework/doc_site/src/views/info_button/info_button_example.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { renderToHtml } from '../../services';
+
+import {
+  GuideDemo,
+  GuidePage,
+  GuideSection,
+  GuideSectionTypes,
+  GuideText,
+} from '../../components';
+
+import Example from './info_button';
+
+const basicSource = require('!!raw!./info_button');
+const basicHtml = renderToHtml(Example);
+
+export default props => (
+  <GuidePage title={props.route.name}>
+    <GuideSection
+      title="Info Button"
+      source={[{
+        type: GuideSectionTypes.JS,
+        code: basicSource,
+      }, {
+        type: GuideSectionTypes.HTML,
+        code: basicHtml,
+      }]}
+    >
+      <GuideText>
+        Is just button with an info icon, used for a keyboard-accessible
+        trigger for helpful inline content. For example, use it as a tooltip
+        trigger. This styles the button to have no background color or border.
```

Comment:
I think the first sentence is missing a reference to "This", and I think the last sentence may not be necessary since the demo itself exhibits these characteristics. How about changing this to:

>  This is just button with an info icon, used for a keyboard-accessible trigger for helpful inline content. For example, use it as a tooltip trigger.